lib/create-organization-api.js

```diff
@@ -111,6 +116,56 @@ export default function createOrganizationApi ({ http }) {
       .then((response) => wrapSpaceMembershipCollection(http, response.data), errorHandler)
   }
 
+  /**
+   * Gets an Invitation in Organization
+   * @memberof ContentfulOrganizationAPI
+   * @param {string} id - Invitation ID
+   * @return {Promise<OrganizationInvitation>} Promise for a OrganizationInvitation in an organization
+   * @example
+   * const contentful = require('contentful-management')
+   *
+   * client.getOrganizations()
+   * .then((organization) => organization.getOrganizationInvitation('invitation_id'))
+   * .then((invitation) => console.log(invitation))
+   * .catch(console.error)
+   */
+  function getOrganizationInvitation (id) {
+    return http.get('invitations/' + id, {
+      headers
```

Comment:
so this one needs a different header than createOrganizationInvitation?

Reply:
@phoebeschmidt yeah, seems like it.

Reply:
We gonna remove the alpha header at the end I think from both though.

Reply:
@phoebeschmidt sorry, just verified it - we would like to keep invitations in alpha for now.